cluster/userdata-master.yaml

```diff
@@ -420,10 +416,11 @@ write_files:
             version: v19
             kubernetes.io/cluster-service: "true"
         spec:
-          replicas: 1
+          replicas: 2
```

Comment:
the dns setup is to be known a bit flaky. somewhere it was recommend to run two copies and gke also does this. should not harm.